Packages/com.llealloo.audiolink/Runtime/Scripts/ThemeColorControllerV0.cs

```diff
@@ -16,13 +17,27 @@ public class ThemeColorControllerV0 : MonoBehaviour
     {
         [UdonSynced] private int _themeColorMode;
 
+        [Obsolete("This array will return a copy of the data, causing it to not write any data when trying to write directly to a index. Use " + nameof(GetCustomThemeColors) + " and " + nameof(SetCustomThemeColors) + " instead.", true)]
```

Comment:
hold on why do we need this when you implemented getter and setter correctly?

Reply:
Because if someone was to directly index to set a value, it wouldnt trigger the setter.

So array[2] = false wouldnt trigger the setter for array, annoyingly

Reply:
got it thanks for the clarification